### PR TITLE
en: update config-overview.md for 2.0

### PR DESCRIPTION
**Pull request by toutdesuite**
### What is changed, added or deleted? (Required)

update config-overview.md for 2.0

### Which TiDB Data Migration (DM) version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-dm/pull/29
- Other reference link(s):

### CODE REVIEW

File: en/config-overview.md [hunk 1]
```diff
@@ -10,31 +10,30 @@ This document gives an overview of configuration files of DM (Data Migration).
...
-- `dm-master.toml`: The configuration file of running the DM-master process, including the topology information of the DM cluster and the corresponding relationship between the MySQL instance and DM-worker (must be one-to-one relationship). When you use DM-Ansible to deploy DM, `dm-master.toml` is generated automatically. Refer to [DM-master Configuration File](dm-master-configuration-file.md) to see more details.
-- `dm-worker.toml`: The configuration file of running the DM-worker process, including the upstream MySQL instance configuration and the relay log configuration. When you use DM-Ansible to deploy DM, `dm-worker.toml` is generated automatically. Refer to [DM-worker Configuration File](dm-master-configuration-file.md) to see more details.
+- `dm-master.toml`: The configuration file of running the DM-master process, including the topology information and the logs of the DM-master and . For more details, refer to [DM-master Configuration File](dm-master-configuration-file.md).
```
Comment (ran-huang) [inline en/config-overview.md:13]:
```suggestion
- `dm-master.toml`: The configuration file of running the DM-master process, including the topology information and the logs of the DM-master. For more details, refer to [DM-master Configuration File](dm-master-configuration-file.md).
```

File: en/config-overview.md [hunk 2]
```diff
@@ -10,31 +10,30 @@ This document gives an overview of configuration files of DM (Data Migration).
...
+- `source.toml`: The configuration of the upstream database such as MySQL and MariaDB. For more details, refer to [Upstream Database Configuration File](source-configuration-file.md).
 
 ## DM replication task configuration
 
 ### DM task configuration file
 
-When you use DM-Ansible to deploy DM, you can find the following task configuration file template in `<path-to-dm-ansible>/conf`:
-
-- `task.yaml.exmaple`: The standard configuration file of the data replication task (a specific task corresponds to a `task.yaml`). For the introduction of the configuration file, see [Task Configuration File](task-configuration-file.md).
+`task.yaml` is the configuration file for the data replication task . For more details, refer to [Task Configuration File](task-configuration-file.md).
```
Comment (ran-huang) [inline en/config-overview.md:21]:
```suggestion
`task.yaml` is the configuration file for the data replication task. For more details, refer to [Task Configuration File](task-configuration-file.md).
```

**Comment by toutdesuite** [comment]
@GMHDBJD PTAL. Thanks~

**Comment by GMHDBJD** [comment]
LGTM

**Pull request merged**